### PR TITLE
REFACTOR ALL THE THINGS

### PR DESCRIPTION
**Pull request by hyperdriveguy**
See #11.

### CODE REVIEW

**Comment by nephitejnf** [comment]
Looks good so far to my fire ridden brain, would you rather I wait before a merge?

**Comment by hyperdriveguy** [comment]
Yeah, this is just the beginning. I'm hoping to fix a bunch of issues as they come up and some of the existing ones as well. I also want to wait so I can test for any possible regressions as I go along as well.

**Comment by nephitejnf** [comment]
> I'm hoping to fix a bunch of issues as they come up and some of the **existing ones** as well.

Heh

File: src/text.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        self.yellow = '\033[93m'
+        self.red = '\033[91m'
+
+
+class TerminalText():
+    """
+    Text that gets output to the terminal.
+
+    All the text that stays constant are in one method.
+    Text the more reusable pieces of text are methods.
```
Comment (hyperdriveguy) [inline src/text.py]: Oops, that's a typo

Comment (hyperdriveguy) [reply src/text.py]: Fiexed

**Comment by hyperdriveguy** [comment]
Help output isn't as pretty from argparse, but it's slightly less code and is properly supporting short options as well as allowing us to drop `-i` and `-o` as options

*As I said, it has only begun...*

**Comment by hyperdriveguy** [comment]
I'll consider this ready for review once the new implementation is at least on par with the previous feature set.

# What's left
## Features
- [x] Reimplement `configparser`
- [x] File write
- [x] Reimplement `parity_check` (Wasn't orginally planning on it, but it's a lot more compact than the original implementation.)
- [ ] ~~Ignore notes that are part of a chord and throw a warning (replaces/coincides with the purpose of `parity_check`)~~
  (I can save this for a future PR if you want.)

## Regressions
- [x] Tempo is output twice
## New bugs
- [ ] ~~Rest optimizations may cause desync if relying on proper `notetype` timing on a rest~~ (**Won't Fix**)
## Code readability
- [x] Final touches on `process_to_file_store()`

**Comment by hyperdriveguy** [comment]
Do you want me to overwrite the contents of `muse2pokecrystal.py` with that of `refactor.py`?
I'll go ahead and do it, but if you want me to revert it, let me know.

**Review by nephitejnf — APPROVED** [review]

**Pull request merged**